### DISCUSSION TITLE
Why many people choose SD1.5 instead of SD2.0?   SD2.0  really unstable?

### DISCUSSION THREAD

**Discussion by Valentina12334tg** [civitai/civitai#367] [Q&A]
As the technology evolves, I find that many people are still choosing SD1.5 over SD2.0. What's the reason?

**Comment by JustMaier** [comment]
Most people in the community prefer the quality of results from the SD1.5 model.
Stability started from the ground up with SD 2.0 and significantly reduced their dataset when training to avoid controversy about the data they used for SD 1.x. It seems to have significantly impacted the quality of the model and most of the community avoid it as a result.

**Reply by WASasquatch** [reply]
Also, it has a strange "look" to people and things that very odd. Not sure if that's object-v or w/e it is they based the new model on or what. Even doing TIs for it, the people look weird. Not in a it's just deformed way, but like, a weird way. Like uncanny valley but unique to 2.x 

In 1.4/1.5 people have a artistic caricature look, so it like avoids any uncanny valley feeling.
